pandas/tests/groupby/aggregate/test_other.py

```diff
@@ -385,10 +385,7 @@ def test_agg_consistency():
     # agg with ([]) and () not consistent
     # GH 6715
     def P1(a):
-        try:
-            return np.percentile(a.dropna(), q=1)
-        except Exception:
-            return np.nan
+        return np.percentile(a.dropna(), q=1)
```

Comment:
Can you just get rid of this function altogether now?

Reply:
This gets passed to `agg` twice.  I think its marginally clearer like this than writing the lambda twice.

Reply:
Ah OK didn't see in diff but makes sense